File: thorium/src/test/benchmark/filter-test/testFilter.R
table <- read.csv("./src/test/benchmark/filter-test/data.csv", stringsAsFactors = FALSE)
table <- subset(table, department == "INFO" & "age" >= 20)
write.csv(table, "./src/test/benchmark/filter-test/R-filter.csv", row.names=FALSE, quote=FALSE)

File: thorium/src/test/benchmark/filter-test/testFilter.py
import pandas as pd
table = pd.read_csv("./src/test/benchmark/filter-test/data.csv")
table = table[(table['department'] == "INFO") & (table['age'] >= 20)]
table.to_csv("./src/test/benchmark/filter-test/Python-filter.csv",index=False)
